apps/docs/src/routes/core/audio-listener.md

```diff
@@ -16,28 +16,28 @@ In most cases, the listener component is a child of the camera, so the 3D transf
 
 ```svelte
 <script>
-  import { Canvas, PerspectiveCamera, AudioListener } from '@threlte/core'
+  import { T, Canvas, AudioListener } from '@threlte/core'
 </script>
 
 <Canvas>
-  <PerspectiveCamera position={{ x: 3, y: 3, z: 3 }} lookAt={{ x: 0, y: 0, z: 0 }}>
+  <T.PerspectiveCamera position={[3, 3, 3]} lookAt={[0, 0, 0]}>
     <AudioListener />
-  </PerspectiveCamera>
+  </T.PerspectiveCamera>
 </Canvas>
 ```
 
 You may pass an `id` to the `<AudioListener>` component in order to connect `<Audio>` and `<PositionalAudio>` components to specific `<AudioListener>` components:
 
 ```svelte
 <script>
-  import { Canvas, PerspectiveCamera, AudioListener, Audio } from '@threlte/core'
+  import { T, Canvas, AudioListener, Audio } from '@threlte/core'
 </script>
 
 <Canvas>
-  <PerspectiveCamera position={{ x: 3, y: 3, z: 3 }} lookAt={{ x: 0, y: 0, z: 0 }}>
+  <T.PerspectiveCamera position={[3, 3, 3]} lookAt={[0, 0, 0]}>
```

Comment:
Because we're talking about a camera that's also receiving audio (likely the scenes' default camera), add `makeDefault`